Sources/UtiliKit/Container/ContainerViewController.swift

```diff
@@ -9,6 +9,12 @@ import UIKit
 
 open class ContainerViewController: UIViewController {
     
+    // MARK: Subtypes
+    public enum PostTransitionBehavior {
+        case removeAllNonVisibleChildren
```

Comment:
Could `.removeAllNonVisibleChildren` be a static let that calls `.removeAllNonVisibleChildrenExcept([])` so we don't have to handle it in switches and stuff?

Reply:
Done, good call